app/models/consultation.rb

```diff
@@ -87,6 +91,38 @@ def add_neighbour_addresses!(addresses, geojson)
     raise AddNeighbourAddressesError, e.message
   end
 
+  def polygon_fill_colour
+    "#{polygon_colour}20"
+  end
+
+  def concatenate_geojsons(*geojsons)
```

Comment:
Need this to represent one geosjon FeatureCollection with all the geosjon features (planning application red line boundary, consultation boundary etc) so we can present this on one map.

If the column type is polygon_geojson we insert the polygon colour so that the map component can distinguish the type of feature